gallery/src/components/NxTabs/NxTabsModalExample.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+const NxTabsModalExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState<number>(0);
```

Comment:
Explicit type parameter not needed here
```suggestion
  const [activeTabId, setActiveTabId] = useState(0);
```